crates/holochain_conductor_api/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+The `AddRecords` admin api method has been changed to `GraftRecords`, and the functionality has changed accordingly. See the docs for that method to understand the changes.
```

Comment:
feels like this should have the `**BREAKING CHANGE**` callout, no?

Reply:
I updated it locally but will wait to push until I get more feedback.